_translations/da.yml

```diff
@@ -202,7 +202,7 @@ da:
     disclosuretxt: "Hvis du finder en sårbarhed relateret til Bitcoin, kan ikke-kritiske sårbarheder sendes pr. email på engelsk til en af kerneudviklerne eller sendes til den private \"bitcoin-security\"-postliste, der er nævnt herover. Et eksempel på en ikke-kritisk sårbarhed kan være mulighed for et denial-of-service-angreb på en tjeneste, som ville være meget dyr at udføre. Kritiske sårbarheder, som er for sensitive til ukrypteret email, bør sendes til en eller flere af kerneudviklerne krypteret med deres PGP-nøgle(r)."
     involve: "Bliv involveret"
     involvetxt1: "Bitcoin er frit programmel og enhver udvikler kan bidrage til projektet. Alt hvad du behøver er i <a href=\"https://github.com/bitcoin/bitcoin\">GitHub-arkivet</a>. Sørg venligst for at læse og følge udviklingsprocessen, der er beskrevet i README-filen, samt at bidrage med kode i god kvalitet og at respektere alle retningslinjer."
-    involvetxt2: "Diskussion om udviklingen foregår på GitHub og postlisten <a href=\"https://lists.linuxfoundation.org/mailman/listinfo/bitcoin-dev\">bitcoin-dev</a> hos Sourceforge. Mindre formel diskussion on udviklen sker på irc.freenode.net #bitcoin-dev (<a href=\"#\" onclick=\"freenodeShow(event);\">netudgave</a>, <a href=\"http://bitcoinstats.com\">logbøger</a>)."
+    involvetxt2: "Diskussion om udviklingen foregår på GitHub og postlisten <a href=\"https://lists.linuxfoundation.org/mailman/listinfo/bitcoin-dev\">bitcoin-dev</a> hos Sourceforge. Mindre formel diskussion on udviklen sker på irc.libera.chat #bitcoin-dev (<a href=\"#\" onclick=\"freenodeShow(event);\">netudgave</a>, <a href=\"http://bitcoinstats.com\">logbøger</a>)."
```

Comment:
<a href=\"#\" onclick=\"freenodeShow(event);\"> links to freenode so, <a href=\"https://web.libera.chat/#bitcoin-core-dev\"> will be right.

This issue exists on next files.
_translations/el.yml 
_translations/es.yml
_translations/fa.yml
_translations/fr.yml
_translations/hi.yml
_translations/ko.yml
_translations/pl.yml
_translations/ro.yml
_translations/ru.yml 
_translations/sl.yml
_translations/uk.yml
_translations/zh_TW.yml

Reply:
Thanks!